nectar/src/config/settings.rs

```diff
@@ -193,6 +195,59 @@ impl Default for KrakenApiHost {
     }
 }
 
+#[derive(Copy, Clone, Debug, PartialEq)]
+pub struct FeeStrategies {
+    pub bitcoin: BitcoinFeeStrategy,
+}
+
+impl From<file::FeeStrategies> for FeeStrategies {
+    fn from(file: file::FeeStrategies) -> Self {
+        Self {
+            bitcoin: file
+                .bitcoin
+                .map_or_else(BitcoinFeeStrategy::default, BitcoinFeeStrategy::from),
+        }
+    }
+}
+
+impl Default for FeeStrategies {
+    fn default() -> Self {
+        Self {
+            bitcoin: BitcoinFeeStrategy::default(),
+        }
+    }
+}
+
+#[derive(Copy, Clone, Debug, PartialEq)]
+pub enum BitcoinFeeStrategy {
+    SatsPerByte(bitcoin::Amount),
+    BitcoindEstimateSmartfee(EstimateMode),
+}
+
+const DEFAULT_BITCOIN_STATIC_FEE_SAT: u64 = 10;
+
+/// Defaults to static fee mode
+/// Default value for static mode is 10 sat per byte
+impl From<file::BitcoinFee> for BitcoinFeeStrategy {
+    fn from(file: file::BitcoinFee) -> Self {
+        match file.strategy {
+            None | Some(file::BitcoinFeeStrategy::Static) => Self::SatsPerByte(
+                file.sats_per_byte
+                    .unwrap_or_else(|| bitcoin::Amount::from_sat(DEFAULT_BITCOIN_STATIC_FEE_SAT)),
+            ),
```

Comment:
You should be able to use `map_or_else` here and re-use the `Default` impl from below.